src/macports1.0/reclaim.tcl

```diff
@@ -339,7 +339,7 @@ namespace eval reclaim {
         set fd -1
         try -pass_signal {
             set fd [open $path w]
-            puts $fd $contents
+            ui_msg " $fd $contents"
```

Comment:
How did you test the change?

Reply:
@mojca  just realized my mistake. Sorry about the bogus PR

Reply:
@arghyatiger, thank you for the pull request.
Even if this one didn't quite work out, we do appreciate your efforts.
We are hoping you will submit many more pull requests that will get merged.